pkgs/development/compilers/corretto/mk-corretto.nix

```diff
@@ -58,6 +58,8 @@ jdk.overrideAttrs (finalAttrs: oldAttrs: {
       # Corretto's actual built is triggered via `gradle`.
       gradle --console=plain --no-daemon ${task}
 
+      set -xv
```

Comment:
Is this needed to fix the Darwin build?

Reply:
No, this is needed for me because I don't have access to macos so I rely on `ofBorg` results.

Reply:
Ok then it doesn't need to be in the commit. You can request the help of @NixOS/darwin-maintainers to fix the remaining Darwin issues.